e2e/support/component-webpack.config.js

```diff
@@ -1,29 +1,29 @@
+const fs = require("fs");
```

Comment:
This file should look almost exactly the same as the one in https://github.com/metabase/metabase/pull/51268, with 1 difference from [this PR](https://github.com/metabase/metabase/pull/52127).

https://github.com/metabase/metabase/blob/3bbae3d59546fb3767f62d56024477a8b87c09c2/e2e/support/component-webpack.config.js#L60